lua/nvim-tree/view.lua

```diff
@@ -392,17 +395,10 @@ function M.is_root_folder_visible(cwd)
 end
 
 function M.setup(opts)
-  local options = opts.view or {}
-  M.View.adaptive_size = options.adaptive_size
-  M.View.side = options.side
-  M.View.width = options.width
-  M.View.initial_width = get_size()
-  M.View.height = options.height
-  M.View.hide_root_folder = options.hide_root_folder
-  M.View.preserve_window_proportions = options.preserve_window_proportions
-  M.View.winopts.number = options.number
-  M.View.winopts.relativenumber = options.relativenumber
-  M.View.winopts.signcolumn = options.signcolumn
+  opts = opts or {}
```

Comment:
Let's not do this refactor. It does not appear to be necessary.

Reply:
> Let's not do this refactor. It does not appear to be necessary.

I mean it's not necessary in the sense that you can keep copying all the parameters by hand and hope you didn't forget any of them, or you can make it both explicit and obvious that this is the intended behavior. 

I'll put it back regardless, but I just wanted to point that out.

---

_P.s. notice how calling this merge function notified me that `height`/`weight` are initially missing.._

Reply:
please rebase, i've fixed this issue when height is top/bottom